sites/public/pages/applications/financial/vouchers.tsx

```diff
@@ -125,14 +125,20 @@ const ApplicationVouchers = () => {
             <fieldset>
               <legend className="sr-only">{t("application.financial.vouchers.legend")}</legend>
               <FieldGroup
+                fieldGroupClassName="grid grid-cols-1"
```

Comment:
Anywhere this change happens - in the application, two radio buttons should be displayed vertically (regression, they are currently displayed horizontally)